travel/src/main/java/com/tx/travel/mapper/CostCenterMapperImplementation.java

```diff
@@ -0,0 +1,31 @@
+package com.tx.travel.mapper;
+
+import com.tx.travel.model.CostCenter;
+import com.tx.travel.payload.request.CostCenterRequest;
+import com.tx.travel.payload.response.CostCenterResponse;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CostCenterMapperImplementation {
```

Comment:
```suggestion
public class CostCenterMapper {
```
since there is no interface, it doen't need to have "implementation" in the name